CMakeLists.txt

```diff
@@ -146,8 +148,8 @@ set(RS_LIBS
 catkin_package(
   INCLUDE_DIRS ${RS_INCLUDE_DIRS_LIST} ${SWIPL_INCLUDE_DIRS} ${LIBAPR_INCLUDE_DIRS} ${Caffe_INCLUDE_DIRS} ${EIGEN3_INCLUDE_DIRS} ${OpenCV_INCLUDE_DIRS} ${PCL_INCLUDE_DIRS}
   LIBRARIES ${RS_LIBS}
-  CATKIN_DEPENDS pcl_ros cv_bridge libmongocxx_ros robosherlock_msgs
-  DEPENDS Boost OpenCV APR PCL UIMA ICUUC 
+  CATKIN_DEPENDS pcl_ros cv_bridge libmongocxx_ros robosherlock_msgs uima
+  DEPENDS Boost OpenCV APR PCL ICUUC MongoClientLibrary
```

Comment:
the same here with MongoClientLibrary...I think something went wrong when you merged the master branch into your fork